src/streamlink/plugins/pluzz.py

```diff
@@ -140,6 +150,13 @@ def _get_streams(self):
             elif '.m3u8' in video_url:
                 for stream in HLSStream.parse_variant_playlist(self.session, video_url).items():
                     yield stream
+            elif '.mp4' in video_url:
+                match = self._mp4_bitrate_re.match(video_url)
+                if match is not None:
+                    bitrate = match.group('bitrate')
+                else:
+                    bitrate = video['format']
+                yield bitrate, HTTPStream(self.session, video_url)
```

Comment:
In fact video['format'] is a fallback when the bitrate is not available in the stream URL (the bitrate is not provided by the Pluzz API). In such a case, video['format'] corresponds to the video type (something like 'mp4-dp' for HTTP MP4 streams here).